tez-dag/src/main/java/org/apache/tez/dag/app/DAGAppMaster.java

```diff
@@ -1056,6 +1057,21 @@ private void writePBTextFile(DAG dag) {
     }
   }
 
+  private void writePBJsonFile(DAGPlan dagPB, DAGImpl newDag) {
+    String logFile = logDirs[new Random().nextInt(logDirs.length)] + File.separatorChar + newDag.getID() + "-"
```

Comment:
I think other files like ``TEZ_PB_PLAN_TEXT_NAME `` are in ``tezSysStagingPath``, should we keep these files also there?
https://github.com/apache/tez/blob/5bba1fffdd290ae3242615c23ab3b75c24f15e88/tez-api/src/main/java/org/apache/tez/common/TezCommonUtils.java#L211-L214